src/source/reference/get-feed-import-id-detail.rst

```diff
@@ -1,8 +1,91 @@
 .. index:: GET /feed/import/{id}/detail
+
 .. _get_feed_import_id_detail:
 
-GET /feed/import/{id}/detail
-============================
+.. _get_feed_import_id_detail_v2:
+
+GET /feed/import/{id}/detail V2
+===============================
+
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``api_ro`` or ``console_ro``
+
+ * - Accept
+   - ``application/sellside.feedimportdetail.list-v2+json, application/json``
+
+This endpoint provides a summary of the feed import.
+
+Each feed import object contains the time when it was started
+(``accepted``), the URL of the user's XML document (``url``) and whether the
+document could successfully be processed (``status``). The ``status`` field
+
+===========  ============    =================================================
+Status Code  Status           Description
+===========  ============    =================================================
+0            PENDING         when it is currently being processed status
+1             DONE           when the document was processed successfully status
+2            REJECTED        when it was either not accessible or syntactically incorrect status
+===========  ============    =================================================
+
```

Comment:
please make the API return a readable string for status.

Reply:
API returns strings already, the docs I am going to fix